### PR TITLE
[Afflication Warlock] Fixed comments from previous merge

### PR DESCRIPTION
**Pull request by sMteX**
I fixed most of the issues from previous merge, as well as some style problems. Please review and if it's okay, I can begin working on more modules.

### CODE REVIEW

File: src/CHANGELOG.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export default `
+18-09-2017 - Added Affliction Warlock suppoert by @Chizu.
```
Comment (MartijnHols) [inline src/CHANGELOG.js]: small typo :P

Comment (MartijnHols) [inline src/CHANGELOG.js]: suppoert

File: src/Parser/AfflictionWarlock/Modules/Features/CastEfficiency.js [hunk 2]
```diff
@@ -44,6 +97,180 @@ class CastEfficiency extends CoreCastEfficiency {
...
+      category: SPELL_CATEGORY.UTILITY,
+      getCooldown: haste => 30,
+      isActive: combatant => combatant.hasTalent(SPELLS.GRIMOIRE_OF_SACRIFICE_TALENT.id),
+      noSuggestion: true,
+    },
+    {
+      spell: SPELLS.BANISH,
+      category: SPELL_CATEGORY.UTILITY,
+      getCooldown: haste => null,
+      noSuggestion: true,
```
Comment (MartijnHols) [inline src/Parser/AfflictionWarlock/Modules/Features/CastEfficiency.js]: all these should also have `noCanBeImproved: true,`

File: src/Parser/AfflictionWarlock/Modules/Features/CorruptionUptime.js [hunk 3]
```diff
@@ -7,18 +7,19 @@ import SPELLS from 'common/SPELLS';
...
 
-  suggestions(when){
+  suggestions(when) {
     const corruptionUptime = this.enemies.getBuffUptime(SPELLS.CORRUPTION_DEBUFF.id) / this.owner.fightDuration;
-    if(this.owner.selectedCombatant.hasBuff(SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id)) {
+    if (this.owner.selectedCombatant.hasBuff(SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id)) {
       when(corruptionUptime).isLessThan(0.95)
         .addSuggestion((suggest, actual, recommended) => {
-          return suggest('Your Corruption uptime can be improved. Try to pay more attention to your Corruption on the boss, especially since you have the T20 2-piece set bonus.')
+          return suggest(<span>Your Corruption uptime can be improved. Try to pay more attention to your Corruption on the boss, which is especially important with the <SpellLink id={SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id} children='T20 2-piece set bonus'/>.</span>)
```
Comment (MartijnHols) [inline src/Parser/AfflictionWarlock/Modules/Features/CorruptionUptime.js]: Children should always be specified like so:

`<SpellLink id={SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id}>T20 2-piece set bonus</SpellLink>`

They're kind of a special property that you shouldn't use directly.

File: src/Parser/AfflictionWarlock/Modules/SoulShards/SoulShardDetails.js [hunk 4]
```diff
@@ -49,21 +49,18 @@ class SoulShardDetails extends Module {
...
-            <SoulShardBreakdown
-              shardsGained = {this.soulShardTracker.gained}
-              shardsSpent = {this.soulShardTracker.spent}
-              shardsWasted = {this.soulShardTracker.wasted}
-            />
-          </div>
-        </div>
+        <Tab
+          title='Soul Shard usage breakdown'
+          children={(<SoulShardBreakdown
```
Comment (MartijnHols) [inline src/Parser/AfflictionWarlock/Modules/SoulShards/SoulShardDetails.js]: see previous children comment 😅

**Comment by MartijnHols** [comment]
Thanks, merged! Please still consider the comments I placed.

File: src/Parser/AfflictionWarlock/Modules/Features/CorruptionUptime.js (hunk 3 above)
Comment (sMteX) [inline src/Parser/AfflictionWarlock/Modules/Features/CorruptionUptime.js]: Ah, I see. I'm not very familiar with these custom components so I assumed this was the way to go. Probably should've looked through other modules first, to see if anyone else had the same problem. Will fix.

File: src/Parser/AfflictionWarlock/Modules/Features/CastEfficiency.js (hunk 2 above)
Comment (sMteX) [inline src/Parser/AfflictionWarlock/Modules/Features/CastEfficiency.js]: Will add too.

**Pull request merged**